support/certs/tls.go

```diff
@@ -356,11 +369,16 @@ func ReconcileSignedCert(
 		secret.Data[caKey] = append([]byte(nil), ca.Data[opts.CASignerCertMapKey]...)
 	}
 
+	certValidity, err := time.ParseDuration(os.Getenv(CertificateValidityEnvVar))
```

Comment:
Can we check whether there's a value, and if not, then use the default. Otherwise try parsing. If there's an error parsing throw the error to prevent things from progressing if an invalid value is passed.